support/json/json.go

```diff
@@ -0,0 +1,49 @@
+package json
+
+import (
+	"encoding/json"
+
+	"github.com/bytedance/sonic"
```

Comment:
So sorry, I missed one thing: the warning is written in the `init` method, the helper method can't remove the warning, because the sonic package will still be complied If it's imported.

Reply:
I pulled the sonic package and tested this situation locally. It seems that we need to remove it totally if we want to remove the warning message.

Reply:
> So sorry, I missed one thing: the warning is written in the `init` method, the helper method can't remove the warning, because the sonic package will still be complied If it's imported.

In this case, maybe we need to write two files `json_x86.go` and `json.go`, and then solve it through build tag(similar to sonic).

Reply:
Oh, that's a great idea!

Reply:
Em, but one question, we need to change the build tag following `bytedance/sonic` in the future.

Reply:
> Em, but one question, we need to change the build tag following `bytedance/sonic` in the future.

I think only need use this, not need to add version:
```
// +build !amd64
// +build amd64
```

Reply:
> Yes, I think so. One more question, I checked our use cases, and we don't have a large data that using `sonic`, based on the benchmark, `sonic` may not bring us a significant improvement, it may be most used in the business logic, what do you think about this, please? Anyway, I think adding the `json_x86.go` file is also a good way.
> 
> <img alt="image" width="1130" src="https://user-images.githubusercontent.com/24771476/272006441-2c121e4b-3686-4046-b809-f51c126d5040.png">

In logging, sonic may not have an obvious effect, but in AES encryption, if the user loops through large amounts of data, the effect will be seen.

In addition, I think it is also a good idea to provide json helper functions for users to choose.

Reply:
Okay, let's add the `json_x86.go` file first, maybe just a few users want to choose it, so I think we don't need to add the helper function for now.

Reply:
> Okay, let's add the `json_x86.go` file first, maybe just a few users want to choose it, so I think we don't need to add the helper function for now.

Completed, I will optimize fiber driver json usage in another PR.